docs/dev/batch/connectors.md

```diff
@@ -217,4 +217,21 @@ The example shows how to access an Azure table and turn data into Flink's `DataS
 
 This [GitHub repository documents how to use MongoDB with Apache Flink (starting from 0.7-incubating)](https://github.com/okkam-it/flink-mongodb-test).
 
+## Hive Connector
```

Comment:
maybe there should be an additional cross page link with https://github.com/apache/flink/blob/master/docs/dev/table/hive/read_write_hive.md except that with catalog.md?